### DISCUSSION TITLE
How to cite Quarto

### DISCUSSION THREAD

**Discussion by Lightbridge-KS** [quarto-dev/quarto-web#203] [Ideas]
Hi, I'm enjoy using Quarto so far. I thinking about citing Quarto in my research paper.

I might suggest including **citation guide for Quarto itself** may be in the web docs or CLI, such as one paragraph of text or BibTeX entry, so that the user knows how to cite the correct way. 

For example, `R` language provide citation as:

```
#> citation()

To cite R in publications use:

  R Core Team (2022). R: A
  language and environment for
  statistical computing. R
  Foundation for Statistical
  Computing, Vienna, Austria. URL
  https://www.R-project.org/.

A BibTeX entry for LaTeX users is

  @Manual{,
    title = {R: A Language and Environment for Statistical Computing},
    author = {{R Core Team}},
    organization = {R Foundation for Statistical Computing},
    address = {Vienna, Austria},
    year = {2022},
    url = {https://www.R-project.org/},
  }
```

Thanks a lot

**Comment by jjallaire** [comment]
We have a citation file on the git repo that enables you get an APA or BibTeX citation here on the main repo page (https://github.com/quarto-dev/quarto-cli.git):

<img width="364" alt="Screen Shot 2022-06-04 at 11 37 35 AM" src="https://user-images.githubusercontent.com/104391/172012929-b4b16036-d057-478b-b8e3-34aaa87c0834.png">

**Reply by Lightbridge-KS** [reply]
Oh, I didn’t see that, Thank you !
